File: service-beacons-go/container/BeaconsProcess.go
package container

import (
	factory "github.com/nov-pocs/samples/service-beacons-go/build"
	cproc "github.com/pip-services3-go/pip-services3-container-go/container"
	gbuild "github.com/pip-services3-go/pip-services3-grpc-go/build"
	rbuild "github.com/pip-services3-go/pip-services3-rpc-go/build"
)

type BeaconsProcess struct {
	cproc.ProcessContainer
}

func NewBeaconsProcess() *BeaconsProcess {
	c := &BeaconsProcess{
		ProcessContainer: *cproc.NewProcessContainer("beacons", "Beacons microservice"),
	}

	c.AddFactory(factory.NewBeaconsServiceFactory())
	c.AddFactory(rbuild.NewDefaultRpcFactory())
	c.AddFactory(gbuild.NewDefaultGrpcFactory())

	return c
}

File: service-beacons-dotnet/src/Service/Container/BeaconsProcess.cs
using Nov.MaxSamples.Beacons.Build;
using PipServices3.Container;
using PipServices3.Rpc.Build;

namespace Nov.MaxSamples.Beacons.Container
{
    public class BeaconsProcess : ProcessContainer
    {
        public BeaconsProcess()
            : base("beacons", "Beacons microservice")
        {
            _factories.Add(new DefaultRpcFactory());
            _factories.Add(new BeaconsServiceFactory());
        }
    }
}
